### PR TITLE
Switched to a superior ide and fixed a few bugs.

### PR DESCRIPTION
**Pull request by IvanVolosyuk**

### CODE REVIEW

File: .ccls [hunk 1]
```diff
@@ -0,0 +1,260 @@
+clang
+
+%c -Wno-frame-address -std=gnu99 -Wno-old-style-declaration -Os -mlongcalls -ffunction-sections -fdata-sections -Wno-error=unused-function -Wno-error=unused-variable -Wno-error=deprecated-declarations -Wno-unused-parameter -Wno-sign-compare -ggdb -freorder-blocks -Wwrite-strings -fstack-protector -fstrict-volatile-bitfields -Wno-error=unused-but-set-variable -fno-jump-tables -fno-tree-switch-conversion -MMD
+%cpp -Wno-frame-address -std=gnu++11 -fexceptions -fno-rtti -Os -mlongcalls -ffunction-sections -fdata-sections -Wno-error=unused-function -Wno-error=unused-variable -Wno-error=deprecated-declarations -Wno-unused-parameter -Wno-sign-compare -ggdb -freorder-blocks -Wwrite-strings -fstack-protector -fstrict-volatile-bitfields -Wno-error=unused-but-set-variable -fno-jump-tables -fno-tree-switch-conversion -MMD
+
+-I/home/dev/kvs/kvs/include
```
Comment (zhsh) [inline .ccls:6]: I don't think this path should be in the shared code.

**Review by zhsh — COMMENTED** [review]
Maybe .ccls should not be commited

File: .ccls (hunk 1 above)
Comment (IvanVolosyuk) [reply .ccls]: Thanks removed.

**Comment by IvanVolosyuk** [comment]
I managed vscode work with nano33ble.

**Pull request merged**